sageresearch-app-sdk/src/androidTest/java/org/sagebionetworks/research/sageresearch/viewmodel/RoomScheduledActivityTests.kt

```diff
@@ -86,21 +83,27 @@ class RoomScheduledActivityTests: RoomTestHelper() {
     @Test
     fun insert_test() {
         activityDao.clear()
-        val first = testResourceMap[activityList]!!.first()
-        activityDao.upsert(listOf(first))
-        assertTaskContains(arrayOf(first.guid), getValue(
-                activityDao.all()))
+        testResourceMap[activityList]?.first()?.let { first ->
```

Comment:
Removing the !! here could cause the test to pass instead of failing with a null pointer. I think in general we should leave the !! for tests so they fail early and don't erroneously pass.

Reply:
Revert it back